### PR TITLE
map-extent updates for hidden, disabled, checked, label attributes

### LINKED ISSUE #254: Update map-extent spec after map-extent custom element development
Documenting https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/887

### PR DESCRIPTION
**Pull request by prushforth**

### CODE REVIEW

**Review by yhy0217 — COMMENTED** [review]
Looks great to me! Just have some ideas about the changes that may make it better:
1. add the information about the default value of extent.checked when the checked attribute is not supplied, as what we did for `label` 
2. replace the closing backslash for map-link with the closing tags in index.html and map-1.html
3. add checked attribute for map-extent in map-1.html and mapml-document.xml

**Review by yhy0217 — APPROVED** [review]

**Review by AliyanH — COMMENTED** [review]
projection web IDL for mapml-viewer is not 'readonly' anymore right?
image

**Review by AliyanH — COMMENTED** [review]
Should we be moving our elements up here when they are working as custom elements?

image

File: spec/index.html [hunk 1]
```diff
@@ -873,6 +859,7 @@ <h4>The <code>&lt;<dfn id="the-layer-element">layer</dfn>&gt;</code> element</h4
           if the layer is not not <a href="#attr-layer-hidden"><code>hidden</code></a>. The map author's <a href="#the-title-element"><code>title</code></a> element takes precedence over <code>label</code>, if available.</dd>
```
Comment (AliyanH) [inline spec/index.html]: the label is not linked to the hidden attribute anymore

**Review by AliyanH — COMMENTED** [review]
Something we might be able to add 
image

**Comment by AliyanH** [comment]
Looks good to me! Seems to have everything from the map-extent PR! @prushforth Should https://github.com/Maps4HTML/MapML/issues/247 be tackled here also? If you think that will interfere with the GeoServer work.

**Comment by prushforth** [comment]
> Should we be moving our elements up here when they are working as custom elements?

One day, as we iterate closer to an [HTML Standard](https://html.spec.whatwg.org/multipage/) PR, we should just treat all the elements as HTML elements, regardless of whether they're new (from MapML) or existing (updates to the spec).  That section (3) is due to merging an earlier separate spec that we developed that separated the content we would expect in an HTML document type from the content that we might expect in a MapML document type.  The line has blurred a bit with the concept of local and remote content, but I don't think custom / not really implemented is a good way to spec them out.

**Comment by prushforth** [comment]
> Should #247 be tackled here also? If you think that will interfere with the GeoServer work.

It wouldn't interfere, but it's related to `<map-feature>` fwiw.  We will get to it eventually.

File: spec/index.html (hunk 1 above)
Comment (prushforth) [reply spec/index.html]: Yes, there's the "not not" which is a typo, but the concept of layers being hidden due to not having a label is a confusion with the old implementation of `map-extent`.  Layers were never hidden for that reason.

**Review by AliyanH — APPROVED** [review]

**Pull request merged**